axolotl-web/src/pages/Register.vue

```diff
@@ -51,19 +51,37 @@ import WarningMessage from "@/components/WarningMessage";
 import { mapState } from "vuex";
 
 export default {
-  name: "register",
+  name: "RegisterPage",
   components: {
     VueTelInput,
     WarningMessage,
   },
-  props: {
-    msg: String,
+  data() {
+    return {
+      phone: "",
+      infoPage: true,
+    };
+  },
+  computed: mapState([
+    "gui",
+    "ratelimitError",
+    "registrationStatus",
+    "captchaToken",
+    "captchaTokenSent",
+  ]),
+  mounted() {
+    var userLang = navigator.language || navigator.userLanguage;
+    this.$language.current = userLang;
+    if (this.captchaToken != null && !this.captchaTokenSent) {
+      this.$store.dispatch("sendCaptchaToken");
+    }
   },
   methods: {
     requestCode() {
       this.$store.dispatch("requestCode", this.phone.replace(/\s/g, ""));
     },
     updatePhone(e) {
+      if(typeof e =="string")
```

Comment:
I believe this could also be a good candidate for the strict `===` check.

Reply:
You are right, as always :)